pkg/stub/stub.go

```diff
@@ -401,6 +401,7 @@ func (stub *stub) Start(ctx context.Context) (retErr error) {
 
 	log.Infof(ctx, "Started plugin %s...", stub.Name())
 
+	stub.started = true
```

Comment:
Line 398 is still working? We can fix it in the follow-up.

```
if err = <-stub.cfgErrC; err != nil {
		return err
	}

```

Reply:
In my understanding and testing, is not affected by this update, what is the concern?

Reply:
I think before we need to let `stub.Configure()` may been called more than once, and these code are must run after `stub.register()`, we can let it remain as it is for the time being, the currently implementation of stub's restart is depend that `stub.register()` is executed every time after restart .